cartography/stats.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, client: StatsClient, prefix: str = None):
         self._client = client
         self._scope_prefix = prefix
 
-    def get_scoped_stats_client(self, scope: str):
+    def get_stats_client(self, scope: str) -> 'ScopedStatsClient':
```

Comment:
Added the type hint as a string. See https://stackoverflow.com/questions/33533148/how-do-i-type-hint-a-method-with-the-type-of-the-enclosing-class